frb/builds/build_frbs.py

```diff
@@ -204,8 +202,7 @@ def frb_190608():
                         340.05 * units.pc / units.cm**3,
                         z_frb=0.1177805)  # Taken from the SDSS table
     # Error ellipse [REQUIRED]
-    frb190608.set_ee(0.19315, 0.18, 0., 68.) # Statistsical
-    frb190608.set_ee(0.178292, 0.18, 0., 68., stat=False)  # Systematic
+    frb190608.set_ee(0.02, 0.3, 0., 68.) # Statistsical
```

Comment:
0.02 seems much too small.
And, again, keep statistical separate from systematic. 
Looks like we might need to ask Cherie for these directly.